### ISSUE TITLE
Exposing a service for third party packages

### ISSUE THREAD

**Issue by abe33** [thomaslindstrom/color-picker#156]
It would be great if the package could provide a service to allow packages to open a color picker with a specific color, a screen position and a function to callback when the user is done with the picker (by passing either the new color or undefined if cancelled, or something along this line).

Related to abe33/atom-pigments#16

**Comment by steelbrain** [comment]
Bump. This would be really useful

**Comment by steelbrain** [comment]
Just something like this would cover my use case
```
interface ColorPicker {
  openColorPicker({textEditor, bufferPosition})
}
```

**Comment by thomaslindstrom** [comment]
@abe33 This should be available in `color-picker` `2.1.0`!
